nexus/types/src/external_api/params.rs

```diff
@@ -466,6 +466,21 @@ pub struct InstanceCreate {
     /// The disks to be created or attached for this instance.
     #[serde(default)]
     pub disks: Vec<InstanceDiskAttachment>,
+
+    /// Whether to not only create the Instance, but also run it.
+    ///
+    /// If `true`, the instance will be created and started immediately.
+    /// If `false`, the instance will be created but not started.
+    ///
+    /// By default, the instance will be created and started.
+    #[serde(default = "bool_true")]
+    pub start: bool,
+}
+
+// Workaround for https://github.com/serde-rs/serde/issues/368
+#[inline]
```

Comment:
does this mean that the inline is necessary or just that we would have preferred to do `#[default = "true"]`? That serde issue seems more like a feature request than a bug

Reply:
Oh, the comment is meant for the function as a whole not the inline attr. I can expand a bit on the comment.

Reply:
I think you can just omit the comment entirely since serde doesn't let you put in a literal and 368 is just some 6 year old idea that seems unlikely to happen any time soon.